src/Cosmos/Signer.cpp

```diff
@@ -51,10 +53,13 @@ Proto::SigningOutput Signer::signProtobuf(const Proto::SigningInput& input, TWCo
 
         auto output = Proto::SigningOutput();
         const std::string jsonSerialized = buildProtoTxJson(input, serializedTxRaw);
+        auto publicKey = PrivateKey(input.private_key()).getPublicKey(TWPublicKeyTypeSECP256k1);
+        auto signatures = nlohmann::json::array({signatureJSON(signature, Data(publicKey.bytes), coin)});
```

Comment:
```suggestion
        auto signatures = nlohmann::json::array({signatureJSON(signature, publicKey.bytes, coin)});
```
No need to copy Data to Data agian :)